### PR TITLE
[release-4.1] Bug 1802860: bump(*):

### PR DESCRIPTION
**Pull request by adambkaplan**
* Fix symlinks with k8s.io/kubernetes
* Pin github/com/mtrmac/gpgme to v0.1.2

Fixes [CVE-2020-8945](https://access.redhat.com/security/cve/CVE-2020-8945)

### CODE REVIEW

**Comment by adambkaplan** [comment]
/retest

AWS rate limiting

File: glide.lock [hunk 1]
```diff
@@ -1,5 +1,5 @@
 hash: d15c7d65c510a024e73ba72a84be2c57d237cef2ab03d7c87af9ff302284645d
-updated: 2019-09-09T15:25:08.380025912-04:00
+updated: 2020-02-21T15:36:28.140353-05:00
```
Comment (gabemontero) [inline glide.lock]: ah glide .... I remember the days when I preferred it to go mod :-)

Comment (adambkaplan) [reply glide.lock]: yup - once upstream added go.mod compatible tags and got past the growing pains, I don't want to go back.

I have a separate backport PR to 4.2 that is awful right now. Need to pin SHAs everywhere.

**Comment by adambkaplan** [comment]
/hold

Image eco is failing due to EOL php image: 

```
Feb 26 19:08:21.815: INFO: At 2020-02-26 19:06:20 +0000 UTC - event for cakephp-example: {buildconfig-controller } BuildConfigInstantiateFailed: error instantiating Build from BuildConfig e2e-test-cakephp-repo-test-4jl8d/cakephp-example (0): Error resolving ImageStreamTag php:7.3 in namespace openshift: unable to find latest tagged image
```

@gabemontero for the purposes of this PR I'm tempted to override.

**Comment by gabemontero** [comment]
> /hold
> 
> Image eco is failing due to EOL php image:
> 
> ```
> Feb 26 19:08:21.815: INFO: At 2020-02-26 19:06:20 +0000 UTC - event for cakephp-example: {buildconfig-controller } BuildConfigInstantiateFailed: error instantiating Build from BuildConfig e2e-test-cakephp-repo-test-4jl8d/cakephp-example (0): Error resolving ImageStreamTag php:7.3 in namespace openshift: unable to find latest tagged image
> ```
> 
> @gabemontero for the purposes of this PR I'm tempted to override.

I would +1 that @adambkaplan

**Comment by adambkaplan** [comment]
/close

4.1 has reached the end of its lifecycle. This fix does not have sufficient severity to warrant a merge.

**Pull request closed without merging**